app/controllers/auth/omniauth_callbacks_controller.rb

```diff
@@ -1,5 +1,19 @@
 # frozen_string_literal: true
 
+def github_next(resp)
```

Comment:
Put these methods inside of the class, after a protected keyword, just like this:

```ruby
class Auth::OmniauthCallbacksController < Devise::OmniauthCallbacksController

  # code

  protected

  # your methods
end
```